### ISSUE TITLE
ID was missing the 'api-version' element with terraform import for a keyvault secret

### ISSUE THREAD

**Issue by LaurentLesle** [Azure/terraform-provider-azapi#119]
Trying to run a terraform import of a keyvault secret:

```
module.solution.module.mssql_managed_instances["sqlmi1"].azapi_resource.sqlmi_admin_password[0]: Importing from ID "/subscriptions/xxx/resourceGroups/caf-rg-sqlmi-re1-ljr/providers/Microsoft.KeyVault/vaults/caf-kv-sqlmirg1-cwf/secrets/caf-sql-lz-sql-mi-txp-password"...
Releasing state lock. This may take a few moments...
Terraform import return code: 1
Terraform returned errors:
╷
│ Error: parsing Resource ID "/subscriptions/xxx/resourceGroups/caf-rg-sqlmi-re1-ljr/providers/Microsoft.KeyVault/vaults/caf-kv-sqlmirg1-cwf/secrets/caf-sql-lz-sql-mi-txp-password": ID was missing the 'api-version' element
```

**Comment by ms-henglu** [comment]
Hi @LaurentLesle ,

Thank you for taking time to report this issue!

It's required to provide `api-version` as a query parameter, for example
```
terraform import azapi_resource.secret "/subscriptions/85b3dbca-5974-4067-9669-67a141095a76/resourceGroups/henglu525/providers/Microsoft.KeyVault/vaults/henglu525/secrets/henglu525secret?api-version=2021-11-01-preview"
```

Ref: https://registry.terraform.io/providers/Azure/azapi/latest/docs/resources/azapi_resource#import

**Comment by LaurentLesle** [comment]
No chances you can retrieve the api-version from the state? That would provide a consistent way of doing import with Terraform

**Comment by ms-henglu** [comment]
Thanks! We'll think about providing a default api-version(maybe the latest api-version) when importing resources into azapi provider.

**Comment by LaurentLesle** [comment]
yes if not provided explicitly using the latest make sense.

**Comment by ms-henglu** [comment]
Hi @LaurentLesle ,

This feature has been released in v0.4.0, would you please give it a try? Thanks!
